model-optimizer/mo/back/offline_transformations.py

```diff
@@ -1,14 +1,27 @@
-#!/usr/bin/env python3
-
-# Copyright (C) 2018-2021 Intel Corporation
+# Copyright (C) 2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+import argparse
+
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    # parser.add_argument("--apply_low_latency", action="store_true")
+    # parser.add_argument("--apply_pruning", action="store_true")
+    parser.add_argument("--path_to_model")
+    args = parser.parse_args()
+    path_to_model = args.path_to_model
+
     try:
-        from openvino.inference_engine import IECore # pylint: disable=import-error
-        from openvino.offline_transformations import ApplyMOCTransformations, CheckAPI # pylint: disable=import-error
+        from openvino.inference_engine import IECore, read_network, read_network_without_extensions # pylint: disable=import-error
+        from openvino.offline_transformations import ApplyMOCTransformations, GenerateMappingFile, CheckAPI # pylint: disable=import-error
     except Exception as e:
         print("[ WARNING ] {}".format(e))
         exit(1)
 
-    CheckAPI()
\ No newline at end of file
+    CheckAPI()
+
+    net = read_network_without_extensions(path_to_model + "_tmp.xml", path_to_model + "_tmp.bin")
```

Comment:
Should we remove the `_tmp` files after serialization?

Reply:
they will be removed.